easybuild/easyconfigs/x/xtb/xtb-6.3.0-foss-2019a-pre.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'MesonNinja'
+
+name = 'xtb'
+version = '6.3.0'
+
+homepage = 'https://xtb-docs.readthedocs.io'
+description = """Semiempirical Extended Tight-Binding Program Package"""
```

Comment:
This description is inconsistent to the Intel build
https://github.com/easybuilders/easybuild-easyconfigs/blob/c2cd3a9bcd774928a913cccf4c28241a7ec699a6/easybuild/easyconfigs/x/xtb/xtb-6.2.2-hotfix-intel-2017b-Python-3.6.3.eb#L8